odl/test/largescale/solvers/nonsmooth/default_functionals_slow_test.py

```diff
@@ -24,20 +24,21 @@
 pytestmark = odl.util.skip_if_no_largescale
 
 stepsize = simple_fixture('stepsize', [0.1, 1.0, 10.0])
-offset = simple_fixture('offset', [False, True])
+linear_offset = simple_fixture('linear_offset', ['none', True])
+quadratic_offset = simple_fixture('quadratic_offset', [False, True])
 dual = simple_fixture('dual', [False, True])
 
 
 func_params = ['l1', 'l2', 'l2^2', 'kl', 'kl_cross_ent', 'const',
                'groupl1-1', 'groupl1-2',
-               'nuclearnorm-1-1', 'nuclearnorm-1-2', 'nuclearnorm-1-inf'
```

Comment:
That this bug has been there for quite some time is scary. How often do we run largescale tests?